android/src/main/java/io/sentry/capacitor/SentryCapacitor.java

```diff
@@ -129,4 +134,23 @@ public void startWithOptions(final PluginCall capOptions) {
 
         capOptions.resolve();
     }
+
+    @PluginMethod
+    public void captureEnvelope(PluginCall call) {
+        String envelope = call.getString("envelope");
+        try {
+            File installation =  new File(sentryOptions.getOutboxPath(), UUID.randomUUID().toString());
+            try (FileOutputStream out = new FileOutputStream(installation)) {
+                out.write(envelope.getBytes(Charset.forName("UTF-8")));
+            } catch (Exception e) {
+                logger.info("Error writing envelope: ", e);
+            }
+        }
+        catch (Exception e) {
+            logger.info("Error reading envelope: ", e);
+        }
+
+        logger.info("Successfully captured envelope")
```

Comment:
This will get called when the exceptions are caught 😊  You might want to move this up to right after the `out.write()`

Reply:
oh, good point. On it!

Reply:
@joshdholtz done!